docs/mp/guides/05_security-oidc.adoc

```diff
@@ -0,0 +1,517 @@
+. Fill the form by adding the realm name, `myRealm` for example.
+. Click on `Create` to create the new realm.
+
+To verify that your realm is created, on the top-left corner where it said `Master` previously
+should be now your realm name or `myRealm` is you followed the example.
+
+To switch from a realm to another, hover the realm name, and the other realm created appear in the dropdown.
+Click on any realm name to change the current realm. Make sure all configuration or modification are saved before changing
+the current realm or be subject to lose your configuration.
+
+==== Create a user
+
+Initially there are no users in a new realm. An unlimited number of user can be created per realm.
+A realm contains resources such as client which can be accessed by users.
+
+To create a new user:
+
+. Open the Keycloak admin console: http://localhost:8080/auth/admin
+. Click on `Users` in the left menu
+. Press `Add user`
+. Fill the form (Username is the only mendatory field) with this value Username: `myUser`
+. Click `Save`
+
+A new user is just created but it needs a password to be able to login. To initialize it, do this:
+
+. Click on `Credentials` at the top of the page, under `Myuser`.
+. Fill `Password` and `Password confirmation` with the user password of your choice.
+. If the `Temporary` field is set to `ON`, the user has to  update password on next login. Click `ON`
+to make it `OFF` and prevent it.
+. Press `Set Password`.
+. A pop-up window is popping off. Click on `Set Password` to confirm the new password.
+
+To verify that the new user is created correctly:
+
+. Open the Keycloak account console: http://localhost:8080/auth/realms/myrealm/account.
+. Login with `myuser` and password chosen earlier.
+
+You should now be logged-in to the account console where users can manage their accounts.
+
+==== Create a Client
+
+To create your first client:
+
+. Navigate to the left menu, into configure section, click on `Clients`. This window displays a table with every client
+from the realm.
+. Click on `Create`
+. Fill the following:
+. `Client ID` : `myClientID`.
+.. `Client Protocol` : `openid-connect`
+.. `Access type` : `confidential`
+.. `Valid Redirect URIs` : `http://localhost:7987/*`
+. Press `Save`
+
+A new tab named `Credentials` is created. Click on it to access this new tab.
+
+- `Client Authenticator` : `Client ID and Secret`
+- Click on `generate secret` to generate client secret.
+
+Keycloak is now configured and ready. Keep keycloak running on your terminal and open a new tab to
+set up Helidon.
+
+=== Set up Helidon
+
+Use the Helidon MP Maven archetype to create a simple project. It will be used as an example
+to show how to set up Helidon.
+
+[source,bash,subs="attributes+"]
+.Run the Maven archetype
+----
+mvn -U archetype:generate -DinteractiveMode=false \
+    -DarchetypeGroupId=io.helidon.archetypes \
+    -DarchetypeArtifactId=helidon-quickstart-mp \
+    -DarchetypeVersion={helidon-version} \
```

Comment:
I think you should mention here, that they have to replace {helidon-version} with the version of Helidon they want to use.